src/lib/sidebar/EntireScheme.svelte

```diff
@@ -163,6 +165,9 @@
 <br />
 
 <div>
+  {#if errorMessage}
+    <ErrorMessage {errorMessage} />
```

Comment:
Note the styling won't appear till after #274, since this page isn't wired up yet